MovieTime/MovieTime/View/Movies/Cells/MTMovieCollectionViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MTMovieCollectionViewCell.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+class MTMovieCollectionViewCell: UICollectionViewCell, MTViewModelSetupCellProtocol {
+
+    // MARK: Static Properties
+    static let informationHeight: CGFloat = 50.0
+
+    // MARK: Outlets
+    @IBOutlet weak var viewBackground: UIView!
+
+    // MARK: Image outlets
+    @IBOutlet weak var viewImageBackground: UIView!
+    @IBOutlet weak var imageViewFrame: UIImageView!
+    @IBOutlet weak var imageViewMovie: UIImageView!
```

Comment:
I started developing the **Settings screen**, there the user could modify the resolution of the images in the app.
That's why I get the `MTModelMovieDBConfiguration` setting (on the Splash screen) and do not use a default value.
Since it is in the config information that there are the possible values of image's resolution.

For this version I used the first resolution and in the next it is possible to modify it in the Settings screen:
```
let fileSize = configuration.images?.logo_sizes?.first ?? "original"

```